concent_api/gatekeeper/views.py

```diff
@@ -74,4 +78,59 @@ def parse_headers(request):
     if loaded_golem_message is None:
         return gatekeeper_access_denied_response("Token in the 'Authorization' header is not a valid Golem message.")
 
+    if loaded_golem_message is None:
+        return gatekeeper_access_denied_response('Undefined golem message.')
+    assert isinstance(loaded_golem_message, Message)
+
+    # Check if request header contains Concent-Client-Public-Key:
+    if 'HTTP_CONCENT_CLIENT_PUBLIC_KEY' not in request.META:
+        return gatekeeper_access_denied_response('Missing Concent-Client-Public-Key header.')
+
+    # Check ConcentFileTransferToken each field:
+    # -DEADLINE
+    if not isinstance(loaded_golem_message.token_expiration_deadline, int):
+        return gatekeeper_access_denied_response('Wrong type of token_expiration_deadline variable.')
+    current_time = int(datetime.datetime.now().timestamp())
+
+    if current_time > loaded_golem_message.token_expiration_deadline:
+        return gatekeeper_access_denied_response('token_expiration_deadline has passed.')
+
+    # -STORAGE_CLUSTER_ADDRESS
+    if not isinstance(loaded_golem_message.storage_cluster_address, str):
+        return gatekeeper_access_denied_response('Wrong type of storage_cluster_address variable.')
+    cluster_address = URLValidator()
```

Comment:
This should be called `url_validator`. It's not an address.

Reply:
Done